<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>751</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>30</y>
        <width>131</width>
        <height>431</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="find_stu_all">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>显示所有学生信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="find_stu">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>查询学生信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="find_stu_all_2">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>显示所有选课信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="find_sc">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>查询选课信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="find_sc_2">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>学生操作</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="find_sc_3">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>课程操作</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="find_sc_4">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>选课操作</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="grade_sel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>成绩统计</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QTableView" name="tableView_manage">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>581</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>157</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>170</y>
        <width>68</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>学生信息查询系统</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadsys">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>128</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ex">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>128</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
